src/framework/uicomponents/qml/MuseScore/UiComponents/Dropdown.qml

```diff
@@ -184,7 +185,7 @@ Item {
 
         contentWidth: root.popupWidth
         contentHeight: root.height * Math.min(root.count, root.popupItemsCount)
-        padding: 0
+        padding: 1
```

Comment:
Could you explain the reason for this change?

Reply:
Ah, this.
The background rectangle for the Dropdowns (specifically for High-Contrast themes) needed an encompassing border, which is only visible with padding.

I think a more appropriate value would be `ui.theme.borderWidth`, so that it only shows up for High-Contrast themes and not the General Light/Dark ones.

Reply:
Yes, better to replace this magic number with ui.theme.borderWidth

Reply:
Fixed :)